src/components/Geogebra/StudentClient.js

```diff
@@ -0,0 +1,453 @@
+/**
+ * [StudentClient] provides API for managing Geogebra applet for students.
+ * Its public API operates as an event emitter for [listener] argument.
+ */
+
+// Initial state of the Geogebra applet encoded in Base64
+const initialState = require('../../helpers/ggbbase64');
+
+const Consts = {
+    // Geogebra specific options
+
+    // Caption style related to label style
+    // determining how objects' names are displayed in
+    // the applet. Caption style means that objects are
+    // named by their captions.
+    CAPTION_STYLE: 3,
```

Comment:
Is 3 a magic const? Can it be described in the comment?

Reply:
Yeah, it is a magic const - that's why I moved it here. What would you like to add here?